code/game/machinery/vending/vending.dm

```diff
@@ -803,29 +877,29 @@ GLOBAL_LIST_EMPTY_TYPED(total_vending_machines, /obj/structure/machinery/vending
 				S.remove_from_storage(item_to_stock, user.loc)
 
 			qdel(item_to_stock)
-			user.visible_message(SPAN_NOTICE("[user] stocks [src] with \a [R.product_name]."),
-			SPAN_NOTICE("You stock [src] with \a [R.product_name]."))
-			R.amount++
+			user.visible_message(SPAN_NOTICE("[user] stocks the [src] with \a [product.product_name]."),
+			SPAN_NOTICE("You stock the [src] with \a [product.product_name]."))
```

Comment:
the the should be being added by dm based on the properness of the datum - is this being weird - or is it just an easier solution than ensuring all the names of the vending types are \improper?

Reply:
I just figure a vending machine should never be proper; even though many people like the capitalize its name and not override it with \improper so it incorrectly will be considered proper (there is only one uncapitalized vendor name, and one other capitalized vendor that is \improper). But I can restore the `\the` usage and either \improper or uncapitalize all vendor names.